ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+#### Subject of the issue
+Describe your issue here
+
+#### Your environment
+* version/branch
+* which browser and its version
+
+#### Steps to reproduce
+Tell us how to reproduce this issue
```

Comment:
Since we use "describe" in Subject of the issue I would be consistent and use "describe" instead of "tell us"